### PR TITLE
fix: plot mass with labels

### PR DESCRIPTION
**Pull request by lpatiny**

### CODE REVIEW

**Comment by lpatiny** [comment]
@wadjih-bencheikh18 Could you take over this PR ?

Need to remove all the xAxis, yAxis if not necessary (everywhere). By default it is x and y and it should be ok.

Please also check the test cases

File: src/app/components/MeasurementMassPlot.tsx [hunk 1]
```diff
@@ -94,19 +92,19 @@ function MassComponent(props: MeasurementPlotProps) {
         <LineSeries
           data={profile}
           lineStyle={{ stroke: 'green' }}
-          xAxis={xAxis}
-          yAxis={yAxis}
+          xAxis="x"
+          yAxis="y"
```
Comment (targos) [inline src/app/components/MeasurementMassPlot.tsx:96]: xAxis is "x" by default, and yAxis is "y" by default. Please remove these props everywhere unless a different value is passed.

**Comment by wadjih-bencheikh18** [comment]
> Need to remove all the xAxis, yAxis

I removed all axisID as mentioned, and updated tests

File: src/app/components/MeasurementMassPlot.tsx (hunk 1 above)
Comment (wadjih-bencheikh18) [reply src/app/components/MeasurementMassPlot.tsx]: fixed

**Pull request merged**